snuba/datasets/errors_replacer.py

```diff
@@ -58,6 +59,8 @@
 logger = logging.getLogger(__name__)
 metrics = MetricsWrapper(environment.metrics, "errors.replacer")
 
+redis_client = get_redis_client(ClusterFunction.REPLACEMENT_STORAGE)
```

Comment:
This pattern should be copied everywhere that uses `redis_client` from `snuba.redis` and then that variable should be deleted